app/models/game.rb

```diff
@@ -60,8 +63,37 @@ def check
     false
   end
 
+  def make_pieces_for_white_player
```

Comment:
This could be called in private at the bottom to the gameboard model so that the two are kept consistent